indico/modules/events/surveys/templates/display/survey_questionnaire.html

```diff
@@ -49,7 +57,23 @@
     {% endcall %}
 
     <script>
-        // allow deselecting a horizontal radio button unless it's required
-        $('#survey-questionnaire .inline-vcentered input:radio:not([required])').resettableRadioButtons();
+        (function() {
+            'use strict';
+            // allow deselecting a horizontal radio button unless it's required
+            $('#survey-questionnaire .inline-vcentered input:radio:not([required])').resettableRadioButtons();
+            $('.js-save-answers').on('declarative:success', function() {
+                var $form = $(this).closest('form');
+                cornerMessage({
+                    actionLabel: $T.gettext('Submit'),
+                    progressMessage: $T.gettext('Submitting...'),
+                    message: $T.gettext('Your answers have been saved, do not forget to submit when you finish!'),
```

Comment:
> Your answers have been saved, do not forget to submit when you finish!

In any case, this string is translatable, meaning that in other words the sentence can and will be longer, meaning that the corner message will grow bigger. The best is to fix the actual problem and make the corner message look better with 2 lines.

Try this:

```html
<div class="corner-message highlight save-corner-message FLEXROW  F-A-CENTER">
     <SPAN>Do not forget to save your changes! Do not forget to save your changes!</SPAN>
     <a class="corner-message-text" href="#" style="FLEX: 0 0 AUTO;">Save now</a>
</div>
```

I noticed that something is overriding the `display: flex` when you add the class `flexrow` to the `.corner-message` element. The proper way to solve it would be to find out what it is and fix it 
 (probably the order the CSS is loaded, I guess the flex partial needs togo higher up in the `_base.scss` file. The workaround would be to use  `display: flex !important` to the element.